lib/models/application.ts

```diff
@@ -1001,6 +1008,7 @@ const getApplicationModel = function (
 						roles: ['provisioning-api-key'],
 						name: keyName,
 						description: keyDescription,
+						expiry_date: keyExpiryDate,
```

Comment:
Did you try this?
Shouldn't it be v ?
```suggestion
						expiryDate: keyExpiryDate,
```
See: https://github.com/balena-io/open-balena-api/blob/37dee1df959c196af3e5c5d8cda4da8cf8d4acce/src/features/api-keys/lib.ts#L104